setup.py

```diff
@@ -1,25 +1,30 @@
+import os
 import pathlib
 from setuptools import setup
 
+version = os.environ.get('ZEGAMI_SDK_VERSION', '')
```

Comment:
I've addressed this in the next commit - it takes the normal version number and appends '+dev' if installed through github or locally. Otherwise CI should give us the wheel without the '+dev' appended on the main branch.